components/function-controller/config/crd/.gitignore

```diff
@@ -0,0 +1 @@
+crd-serverless.yaml
```

Comment:
This file is not used anywhere and it's copied to the `installation` directory. It doesn't make sens to keep it in git as is transient file.